### DISCUSSION TITLE
Correct terminology throughout

### DISCUSSION THREAD

**Discussion by andrewpatto** [umccr/elsa-data#35] [General]
Interested in @sarahcasauria. @MVidgen, @MatildaAHaas thoughts on terminology in the UI.

Data custodian v data manager v data owner..

PI v CI v researcher

Cases, individuals, patients, specimens, samples, biosamples, artifacts

Releases, datasets, applications (I think these ones are straight forward and we probably already use correctly)..

Interested if anyone has any strong feelings of the meanings of any of these terms that clashes with how we are using them in Elsa Data.. and/or any thoughts on more appropriate words

**Comment by MVidgen** [comment]
No really strong feelings, just suggestions.

**Data custodian v data manager v data owner**
MCRI is currently reviewing its Data Governance, including defining some of these terms. So my response is based on the current thinking at MCRI. The below definitions are cut and pasted from the latest MCRI Data Owner Framework document. Some wording might change before it is approved, but the overall emphasis will remain the same. Australian Genomics currently uses these terms differently.

I’ll check if I can share the whole document, but below are the definition for the terms you are asking about
It is a three-tier system – data owner, data stewards and data custodians. I think data owner or data steward would be reasonable terms to use in Elsa’s context. In this terminology structure, data custodians would be roles like sys admin. I’m not sure data steward is a commonly recognised term, but it is more encompassing of the different roles that might be involved, i.e. data curator, data manager, and data officer. 

“Data owner 
A data owner is concerned with risk and appropriate management of access to data. They are accountable for data governance outcomes and responsible for creating a data governance framework for their data, which is aligned with central policies but also right-sized for the project. Some projects may include multiple data owners, each responsible for a different domain of data, and/or an overarching data owner, responsible for the project as a whole.  

Data Steward
A data steward is responsible for ensuring the data governance framework is properly adhered to.  They are domain experts, who understand the data and ensure artefacts such as data dictionaries are correct, and that data formats for both sharing and storage are correct. They are sometimes known by other role titles, for example data manager. 

Data Custodian
Data custodians know where the data is and how to access it. In many organisations the data custodian is an IT person or a data application support person, someone that is not expected to have domain knowledge but a full understanding of the underlying systems used to manage the data, and how to access it. A specific project may have many data custodians, depending on the different technologies and tools in use.“ – MCRI Data Owner Framework (as of 01/09/2022)

**PI vs CI vs researcher**
PI and CI are very restrictive to a particular type of researcher. There are scenarios where data may be accessed by a person that is not a researcher (i.e. research in commercial settings). To be broader in our terminology, we use the term applicant or data recipient.

**Cases, individuals, patients, specimens, samples, biosamples, artifacts**
We use the term participant rather than patient. As a research project, to AG they are research participants even if in the clinical settings they are patients. Also, the term ‘participant’ encompasses family members that are not patients, but whose data is also in the system.

**Releases, datasets, applications**
Based on the summary you gave I think these are being use appropriately.

**Comment by MatildaAHaas** [comment]
Hi Andrew,

I’m having trouble viewing/accessing this conversation on GitHub. I mentioned this to Christian and he said you may need to add me to the UMCCR organisation?

Many thanks,

Matilda

Matilda Haas
M +61 403 287 727

From: Andrew Patterson ***@***.***>
Date: Wednesday, 31 August 2022 at 11:03 am
To: umccr/elsa-data ***@***.***>
Cc: Matilda Haas ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL][UMCCR] Correct terminology throughout (Discussion #35)
CAUTION:  External Email. Please be cautious with attachments and clicking links


Interested in @sarahcasauria<https://github.com/sarahcasauria>. @MVidgen<https://github.com/MVidgen>, @MatildaAHaas<https://github.com/MatildaAHaas> thoughts on terminology in the UI.

Data custodian v data manager v data owner..

PI v CI v researcher

Cases, individuals, patients, specimens, samples, biosamples, artifacts

Releases, datasets, applications (I think these ones are straight forward and we probably already use correctly)..

Interested if anyone has any strong feelings of the meanings of any of these terms that clashes with how we are using them in Elsa Data.. and/or any thoughts on more appropriate words

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/umccr/discussions/35>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATGF4B7XEO56V3ECHFKMAILV32VMNANCNFSM6AAAAAAQA5JWKQ>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Disclaimer

This e-mail and any attachments to it (the "Communication") are, unless otherwise stated, confidential, may contain copyright material and is for the use only of the intended recipient. If you receive the Communication in error, please notify the sender immediately by return e-mail, delete the Communication and the return e-mail, and do not read, copy, retransmit or otherwise deal with it. Any views expressed in the Communication are those of the individual sender only, unless expressly stated to be those of Murdoch Children’s Research Institute (MCRI) ABN 21 006 566 972 or any of its related entities. MCRI does not accept liability in connection with the integrity of or errors in the Communication, computer virus, data corruption, interference or delay arising from or in respect of the Communication.

**Reply by MatildaAHaas** [reply]
I've got it! I'm in! Sorry for the false start on my part
